internal/cloudrunci/cloudrunci.go

```diff
@@ -384,3 +389,38 @@ func (s *Service) urlCmd() *exec.Cmd {
 	cmd.Dir = s.Dir
 	return cmd
 }
+
+func (s *Service) LogEntries(filter string, find string, maxAttempts int) (bool, error) {
+	ctx := context.Background()
+	client, err := logadmin.NewClient(ctx, s.ProjectID)
+	if err != nil {
+		return false, fmt.Errorf("logadmin.NewClient: %v", err)
+	}
+	defer client.Close()
+
+	preparedFilter := fmt.Sprintf(`resource.type="cloud_run_revision" resource.labels.service_name="%s" %s`, s.version(), filter)
+	fmt.Printf("Using log filter: %s\n", preparedFilter)
+
+	for i := 1; i < maxAttempts; i++ {
+		fmt.Printf("Attempt #%d\n", i)
+		it := client.Entries(ctx, logadmin.Filter(preparedFilter))
+		for {
+			entry, err := it.Next()
+			if err == iterator.Done {
+				break
+			}
+			if err != nil {
+				return false, fmt.Errorf("it.Next: %v", err)
+			}
+			if len(fmt.Sprintf("%v", entry.Payload)) > 0 {
+				fmt.Printf("entry.Payload: %v\n", entry.Payload)
+			}
+			if strings.Contains(fmt.Sprintf("%v", entry.Payload), find) {
+				fmt.Println("SIGTERM log entry: found.")
```

Comment:
Update to use `find`. Consider printing with `Printf` and using `%q` to wrap `find` in quotes.